cmd/flags.go

```diff
@@ -130,6 +130,11 @@ func addAsAddressFlag(cmd *cobra.Command) {
 		"Attestation Service address")
 }
 
+// Lets this command enable Cloud logging
+func addCloudLoggingFlag(cmd *cobra.Command) {
+	cmd.Flags().Bool("cloudLog", false, "cloud logging switch")
+}
+
```

Comment:
`cloudLog` and `asAddr` should be lower case and separated by hypens. For example `cloud-log` and `as-addr`. See the `hash-algo` flag.

Reply:
`as-addr` is not very descriptive either. Consider something like `service-endpoint` or `verifier-endpoint`